crypto/s2n_cbc_cipher_aes.c

```diff
@@ -33,21 +33,21 @@ static uint8_t s2n_cbc_cipher_aes256_available()
 
 static int s2n_cbc_cipher_aes_encrypt(struct s2n_session_key *key, struct s2n_blob *iv, struct s2n_blob *in, struct s2n_blob *out)
 {
-    POSIX_ENSURE_GTE(out->size, in->size);
+    POSIX_ENSURE_GTE((int) out->size, (int) in->size);
```

Comment:
Aren't these the same integer types? We shouldn't need to cast if they are.

In general, we need to be really careful about downcasting sizes, as we can truncate and wrap if the values are too large. When in doubt, I would go with a `uint64_t`/`int64_t` just to be safe.